Tests/NetNewsWireTests/ArticleSorterTests.swift

```diff
@@ -0,0 +1,240 @@
+//
+//  ArticleSorterTests.swift
+//  NetNewsWire
+//
+//  Created by Phil Viso on 9/8/19.
+//  Copyright © 2019 Ranchero Software. All rights reserved.
+//
+
+import Articles
+import Foundation
+import XCTest
+
+@testable import NetNewsWire
+
+class ArticleSorterTests: XCTestCase {
```

Comment:
For some reason, from this point down in the PR all of the indentation looks weird. I'm using whatever the defaults are in Xcode for indentation settings, so I'm not sure why GitHub is showing these huge indentations.